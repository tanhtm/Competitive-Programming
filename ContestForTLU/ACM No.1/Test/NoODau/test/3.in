174 192
3 23 26 35 39 52 53 56 69 78 85 90 92 97 103 105 108 109 110 111 113 123 125 136 147 148 154 157 166 169 173 177 185 186 200 209 222 224 226 226 235 235 239 242 242 251 252 254 275 279 280 291 293 325 325 332 342 356 360 361 370 374 376 382 383 385 387 388 394 395 401 406 417 428 430 436 438 438 443 443 448 448 451 457 472 473 482 486 487 492 496 507 514 514 514 533 544 545 547 552 559 564 581 581 581 585 592 604 616 630 635 641 652 661 665 680 682 683 683 683 693 693 697 702 704 707 722 725 725 726 745 752 769 776 776 789 792 793 795 802 823 826 830 832 837 847 856 860 868 872 874 876 877 882 882 895 898 899 913 914 914 917 935 943 948 956 966 969 977 977 978 980 984 993 
615 769 927 513 879 161 86 79 337 366 596 73 141 94 208 661 464 673 220 50 205 1 875 192 346 29 634 22 490 679 573 104 799 852 968 678 364 54 108 52 419 703 124 911 796 683 571 611 355 142 13 912 143 239 455 488 267 440 509 108 118 433 564 917 636 531 946 351 936 405 402 354 459 877 616 254 560 539 217 266 680 581 177 174 819 983 661 86 423 521 545 892 306 108 160 941 991 105 292 926 509 45 632 319 922 247 925 833 785 141 450 817 721 979 990 539 961 3 976 383 523 873 275 828 980 434 121 970 891 764 248 751 808 879 70 81 125 994 265 262 486 715 78 558 693 67 448 653 421 776 388 944 648 14 123 979 447 595 301 689 358 548 792 518 426 861 950 902 206 215 515 691 281 592 600 973 11 47 977 431 822 716 726 469 729 201 800 528 795 100 216 505 
