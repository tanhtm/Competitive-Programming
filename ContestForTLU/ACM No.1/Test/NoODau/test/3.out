109
133
163
93
154
29
12
11
57
61
108
10
25
14
36
114
85
116
37
6
36
1
152
35
58
4
111
2
90
116
103
16
140
147
168
116
61
8
17
6
74
125
23
159
140
118
103
109
58
25
2
159
25
43
84
90
49
79
93
17
22
76
102
162
112
96
165
58
164
72
72
58
85
153
109
48
102
97
37
49
116
103
32
32
141
173
114
12
74
96
98
156
54
17
29
164
174
16
53
163
93
6
111
54
163
46
163
145
136
25
83
141
127
172
174
97
167
1
169
65
96
151
49
143
172
76
22
169
156
133
46
132
141
154
10
11
23
174
49
49
88
127
10
101
121
9
81
114
74
134
68
165
113
2
22
172
81
108
54
121
59
100
137
96
74
149
166
159
36
37
96
121
52
107
108
169
2
6
169
76
141
127
130
85
131
36
140
96
139
15
37
92
